### PR TITLE
[restinio] Update to 0.6.15

### PR DESCRIPTION
**Pull request by eao197**
RESTinio updated to the latest version.

### CODE REVIEW

File: ports/restinio/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "restinio",
-  "version": "0.6.14",
+  "version": "0.6.15",
   "port-version": 1,
```
Comment (Thomas1664) [inline ports/restinio/vcpkg.json:4]:
```suggestion
  "version": "0.6.15",
```

File: versions/r-/restinio.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
 {
   "versions": [
+    {
+      "git-tree": "07fadf731052fa7900593db32c0515cba8c45138",
+      "version": "0.6.15",
+      "port-version": 0
+    },
+    {
```
Comment (ras0219-msft) [inline versions/r-/restinio.json:8]: PRs should only ever add one version to the database. Can you run:

```
git checkout 762992cd6118b44ef6ddfab6d2410026c96b5122~ -- versions
./vcpkg x-add-version restinio
git commit -m "Remove additional version entry" versions
```

**Review by ras0219-msft — CHANGES_REQUESTED** [review]
Thanks for the update!

File: versions/r-/restinio.json (hunk 2 above)
Comment (eao197) [reply versions/r-/restinio.json:8]: Done.

**Review by FrankXie05 — APPROVED** [review]

**Review by ras0219-msft — APPROVED** [review]

**Pull request merged**